internal/handler/message/handler.go

```diff
@@ -22,36 +23,60 @@ type handler struct{}
 
 func (h *handler) rules(ctx context.Context, bot *telego.Bot, message telego.Message) {
 	log := logger.FromContext(ctx)
+
+	log = log.With(slog.Group("user",
+		slog.String("username", message.From.Username),
+		slog.Int64("id", message.From.ID),
+	))
+
+	log.Info("Handling", slog.String("command", "rules"))
```

Comment:
We don't need such kind of log messages (in other places as well) cos it provides no information. Moreover, in case of multiple requests it will also consume additional resources.

We could keep it in Debug level but still don't see any benefits of doing that.